templator/templator.go

```diff
@@ -108,6 +114,48 @@ func NewDockerFileTemplator(box *packr.Box) *DockerTemplator {
 	return &DockerTemplator{
 		ApplicationDocker: appTemplate,
 		HttpGatewayDocker: httpTemplate,
-		DockerIgnore: ignoreTemplate,
+		DockerIgnore:      ignoreTemplate,
+	}
+}
+
+type DirectoryTemplator struct {
```

Comment:
this provides a very generic templator that we can use for any directory

Reply:
How would this work for bits where each file needs a bit of additional context like here https://github.com/commitdev/commit0/blob/f679a5cb7eedf23c3f08a612a65ba5b7f4fee9f2/generate/proto/generate.go#L69 ?